terraform/cloud-platform-components/resources/cloudwatch-exporter.yaml

```diff
@@ -0,0 +1,81 @@
+# Values file for Helm chart stable/prometheus-cloudwatch-exporter
+# This YAML-formatted file will be called by Terraform.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: prom/cloudwatch-exporter
+  tag: cloudwatch_exporter-0.5.0
+  pullPolicy: IfNotPresent
+
+aws:
+  # The name of a pre-created secret in which AWS credentials are stored. When
+  # set, aws_access_key_id is assumed to be in a field called access_key,
+  # aws_secret_access_key is assumed to be in a field called secret_key, and the
+  # session token, if it exists, is assumed to be in a field called
+  # security_token
+  # Secret deployed from monitoring namespace in environments
+  secret:
+    name: "cloudwatch-prometheus-secret"
+    includesSessionToken: false
```

Comment:
This secret is created in the environments repository. PR to follow. There is a question of whether this should exists as a Terraform resource?

Reply:
Can we not provide the necessary credentials with a role, using `kiam`?

Reply:
Yes, changed to reflect the above comment as this should assume a role via KIAM.